### ISSUE TITLE
add description on service.name key

### ISSUE THREAD

**Issue by andrey-gava** [hazelcast/charts#211]
service.name value is misleading. When deploy with it chart still use templated full name from _helpers.tpl 
It will only work if two condition are met:
1 service.create=false (so its need that service exist at this point)
2 service.name value are set and not null

Examples:
```
1 service.create not set  (default=true)
2 service.name=new-svc-name
Output
service.name= hazelcast.fullname from _helpers.tpl 
```
----------------
```
1 service.create=false
2 service.name not set
Output
service.name=default
```
----------------
```
1 service.create=false
2 service.name=new-svc-name
Output
service.name=new-svc-name
```

**Comment by leszko** [comment]
Good point, would you mind sending a PR to improve the description?

**Comment by andrey-gava** [comment]
Try to add pr #212 but as I new to github I do some mistakes in set email of user. I hope you see the point.

**Comment by leszko** [comment]
Fixed in #213
